contracts/contribute/ContributeManager.sol

```diff
@@ -225,7 +237,8 @@ contract ContributeManager {
         uint256 threshold = IStaking(stakingInstance).threshold();
         // Check for number of agent instances that must be equal to one,
         // since msg.sender is the only service multisig owner
-        if (numAgentInstances != NUM_AGENT_INSTANCES || threshold != THRESHOLD) {
+        if ((numAgentInstances > 0 &&  numAgentInstances != NUM_AGENT_INSTANCES) ||
```

Comment:
If any of the fields are zeros in the staking contract, we need to ignore the check.